src/main/scala/com/databricks/spark/xml/XmlOptions.scala

```diff
@@ -51,6 +51,9 @@ private[xml] class XmlOptions(
   val ignoreSurroundingSpaces =
     parameters.get("ignoreSurroundingSpaces").map(_.toBoolean).getOrElse(false)
   val parseMode = ParseMode.fromString(parameters.getOrElse("mode", PermissiveMode.name))
+  val version = parameters.getOrElse("version", null)
```

Comment:
I would make the version default like `<?xml version="2.0" encoding="UTF-8"?>`.

Reply:
I prefer to make it is a optional , because It is easier for us to merge them together with hdfs merge tool.  otherwise , there will be duplicate headers